jobs/node_info.ts

```diff
@@ -1,20 +1,24 @@
-const { Config } = require('../models');
+import { Telegraf } from "telegraf";
+import { MainContext } from "../bot/start";
+
+import { Config } from '../models';
 const { getInfo } = require('../ln');
-const logger = require('../logger');
+import logger from "../logger";
 
-const info = async bot => {
+const info = async (bot: Telegraf<MainContext>) => {
   try {
     const config = await Config.findOne({});
+    if (config === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program